setup.cfg

```diff
@@ -23,9 +23,17 @@ exclude =
     .scratch,
     _vendor,
     data
-select = E,W,F
-ignore = W504
-
+    # TODO: Remove when fixing flake8 warnings
+    src/pip/*
+# B = bugbear
+# E = pycodestyle errors
+# F = flake8 pyflakes
+# W = pycodestyle warnings
+# B9 = bugbear opinions
+select = B, E, W, F, B9
```

Comment:
generally you don't need this setting

Reply:
Removed in https://github.com/pypa/pip/pull/8398/commits/59b27ad0f401938a0402ab4d408a6092d681c763